firmware/integration/rusefi_config.txt

```diff
@@ -1191,13 +1194,23 @@ uint8_t[4] unusuedsw;
 	uint8_t[IAC_PID_MULT_SIZE] iacPidMultLoadBins;;"Load",   1,   0.0,        0,  500.0,   2
 	uint8_t[IAC_PID_MULT_SIZE] iacPidMultRpmBins;;"RPM",        @@RPM_1_BYTE_PACKING_MULT@@,     0,      0.0,   12000.0,   0	
 
-	int[527] mainUnusedEnd;
+	int[479] mainUnusedEnd;
 
 ! end of engine_configuration_s
 end_struct
 
 engine_configuration_s engineConfiguration;
 
+
+float[AFTERSTART_HOLD_CURVE_SIZE] afterstartHoldTimeBins;;"C",        1,     0,   -100.0,    250.0,  0
```

Comment:
Can we share the bins between all of these? Is there a case where we need different bins for each of these tables?

Reply:
They could share bins for sure..

Reply:
Did share their bins :-)